resources/contents/blog/en/customer-experience-instead-of-features.md

```diff
@@ -0,0 +1,54 @@
+---
+id: 201902181
+title: "Stop thinking in features. Start thinking in customer experience."
+date: 2019-02-18T19:45:00+02:00
+author: Rathes Sachchithananthan
+template: post
+description: Instead of adding more and more features improve the experience the user makes with your product. It will have a bigger impact on your churn.
+categories:
+  - Business
+tags:
+  - Product Development
+  - Customer Experience
+locale: en_US
+---
+
+A common misconception that I notice quite often among people in the business of product development is that the more features their product has the better it will be. But that is absolutely not true.
+Your customer does not need just another feature. Think back at how you made them choosing your product and why they became your customer. They were looking for a solution that would solve their problem and your product seemed to be the best fit. They choose your product and try it in their day to day business. At a certain point they will notice that your product is just not what they were looking for and will quit using it.
+The reason that your product was not their best fit is not only caused by missing feature. You can have all the features the users need and still have a high churn. What you should look for instead is how exactly you are solving the user's problem.
+
+## Listen to your customers
+
+Start listening to them and start understanding them. Ask the right questions and identify the pain point in their lives. Identify what problem they expect to be solved with your product.
+If the expectations are not on par with your product vision, you should consider how you communicate about your product. Apparently, you are targeting the wrong people or trigger false thoughts.
+But if your vision of your product is exactly about solving your customers problem, then focus on exactly that. Instead of thinking what your product needs think about your customer. What do they need, how do you solve their problem.
```

Comment:
```suggestion
But if your vision of your product is exactly about solving your customers' problem, then focus on exactly that. Instead of thinking about what your product needs think about your customer. What do they need, how do you solve their problem.
```